docs/tutorials/readout-error-mitigation-sampler.ipynb

```diff
@@ -0,0 +1,1414 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "70810e0b-08f9-48ae-beb5-8c863819000b",
+   "metadata": {},
+   "source": [
+    "{/* cspell:ignore braket, Zgrm, newcommand, probs, quasis, topten */}\n",
+    "# Readout error mitigation for the Sampler primitive using M3\n",
+    "\n",
+    "*Usage estimate: under one minute on a Heron r2 processor (NOTE: This is an estimate only. Your runtime might vary.)*\n",
+    "\n",
+    "## Background\n",
+    "\n",
+    "Unlike the Estimator primitive, the Sampler primitive does not have built-in support for error mitigation.\n",
+    "Several of the methods supported by the Estimator are specifically designed for expectation values, and hence are not applicable to the Sampler primitive. An exception is readout error mitigation, which is a highly effective method that is also applicable to the Sampler primitive.\n",
+    "\n",
+    "The [M3 Qiskit addon](https://qiskit.github.io/qiskit-addon-mthree/) implements an efficient method for readout error mitigation. This tutorial explains how to use the M3 Qiskit addon to mitigate readout error for the Sampler primitive.\n",
+    "\n",
+    "### What is readout error?\n",
+    "\n",
+    "Immediately before measurement, the state of a qubit register is\n",
+    "described by a superposition of computational basis states,\n",
+    "or by a density matrix.\n",
+    "Measurement of the qubit register into a classical bit register then proceeds in two steps.\n",
+    "First the quantum measurement proper is performed.\n",
+    "This means that the state of the qubit register\n",
+    "is projected onto a single basis state that is characterized\n",
+    "by a string of $1$s and $0$s.\n",
+    "The second step consists of reading the bitstring characterizing this basis state\n",
+    "and writing it into classical computer memory.\n",
+    "We call this step *readout*.\n",
+    "It turns out that readout error is more important than error because it can incorrectly collapse the state.\n",
```

Comment:
```suggestion
    "It turns out that the second step (readout) incurs more error than the first step (projection onto basis states).\n",
```
My original text here was poorly worded and unclear. Your edit changed my intended meaning. I have tried to make it more clear in this suggestion. But I'm still not 100% happy with the syntax.